www/profile/app-profile.less

```diff
@@ -21,8 +21,14 @@
         display: none !important;
     }
 
+    #cp-app-profile-rightside:has(.cp-app-profile-friend-container) #cp-app-profile-link {
```

Comment:
Please add a comment explaining why this needs to be hidden (element was already not visible but adding extra margin to the layout)